### PR TITLE
Translation resource

### PR DESCRIPTION
**Pull request by xueqingxiao**

### CODE REVIEW

**Comment by benjycui** [comment]
cc @visvadw

**Comment by afc163** [comment]
:+1: Awesome translation for book links

File: docs/resource/download.en-US.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+  <span class="resource-card-content">
+    <span class="resource-card-title">Ant Design Library v2.0.1</span>
+    <span class="resource-card-description">A powerful Axure library of Ant Design.</span>
+  </span>
+</a>
+<a target="_blank" href="http://ux.ant.design" class="resource-card">
+  <img src="https://os.alipayobjects.com/rmsportal/yfTqrQuSKcqBDLY.png">
+  <span class="resource-card-content">
+    <span class="resource-card-title">Ant UX</span>
+    <span class="resource-card-description">A series prototype library, help you to organize your page.</span>
```
Comment (visvadw) [inline docs/resource/download.en-US.md]: <span class="resource-card-description">A series prototype library that helps you creating a nice user experience, covering the page structure and user flow within the page.</span>

File: docs/resource/download.en-US.md [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+  <span class="resource-card-content">
+    <span class="resource-card-title">Web Font</span>
+    <span class="resource-card-description">Icon font package for your local reference.</span>
+  </span>
+</a>
+<a target="_blank" class="resource-card disabled">
+  <img src="https://os.alipayobjects.com/rmsportal/CxXbSpIXckHOtWl.png">
+  <span class="resource-card-content">
+    <span class="resource-card-title">Ant Cool</span>
+    <span class="resource-card-description">An Agile tool for sharing design resource.</span>
```
Comment (visvadw) [inline docs/resource/download.en-US.md]: <span class="resource-card-description">An Agile tool for sharing design resources.</span>

File: docs/resource/reference.en-US.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+order: 1
+title: Reference
+---
+
+Below Books has inspired us, saved our time and helped us to overome difficulty when we design component and patterns. If you want to know more about design things, we recommend that you read these awesome books.
```
Comment (visvadw) [inline docs/resource/reference.en-US.md]: Please find below the books that inspired us, saved our time and helped us to overcome difficulties when designing components and patterns. If you want to know more about UI design, we recommend you these awesome books.

**Review by dxps — APPROVED** [review]
Here are my small adjustments.
Indeed, it's a great translation! 👍

**Comment by xueqingxiao** [comment]
@visvadw Thanks for your improvement. @benjycui I've done.

**Pull request merged**